packages/renderer/src/lib/kube/KubernetesEmptyScreen.svelte

```diff
@@ -1,12 +1,57 @@
 <script lang='ts'>
 import { EmptyScreen } from '@podman-desktop/ui-svelte';
 import type { ComponentProps } from 'svelte';
+import { onDestroy, onMount } from 'svelte';
 
 import KubernetesCheckConnection from '/@/lib/ui/KubernetesCheckConnection.svelte';
+import type { IDisposable } from '/@api/disposable';
 
-let { icon, ...restProps }: ComponentProps<EmptyScreen> = $props();
+import { listenResourcePermitted } from '../kube/resource-permission';
+
+interface Props extends ComponentProps<EmptyScreen> {
+  resources: string[];
+  titleEmpty: string[];
+  titleNotPermitted: string[];
+}
+
+let { icon, resources, titleEmpty, titleNotPermitted, ...restProps }: Props = $props();
+
+let resourcesPermitted: boolean[] = $state(resources?.map(r => r !== ''));
+let disposables: IDisposable[] = [];
+let title = $derived.by(() => {
+  let resourcesTitle = '';
+  const lengthOfResources = resources?.length;
+
+  // For each resource select text based on whether the resource is permitted
+  for (let i = 0; i < lengthOfResources; i++) {
+    if (resourcesPermitted[i]) {
+      resourcesTitle += titleEmpty[i];
+    } else {
+      resourcesTitle += titleNotPermitted[i];
+    }
+
+    // Adding new line
+    if (i + 1 < lengthOfResources) resourcesTitle += '\n';
+  }
+  return resourcesTitle;
+});
+
```

Comment:
I don't think this is what we want.

For the version with no resource, I think that we don't want to change the behaviour, so we should still have 'No ingresses or routes', or nothing if there is at least one ingress or one route.

For permissions, I think that we want to display that we don't have permissions for **all** resources, or nothing. If the user has permission for at least one resource type, nothing should be marked.